delegation/cardano-sl-delegation.cabal

```diff
@@ -52,6 +52,7 @@ library
                      , cardano-sl-networking
                      , cardano-sl-util
                      , conduit
+                     , containers
```

Comment:
If you revert all unnecessary changes, this dependency shouldn't be necessary I suppose.

Reply:
Fixed.